### ISSUE TITLE
WIP

### ISSUE THREAD

**Issue by lopestom** [sudo-Atharva/android_device_Motorola_vicky-twrp#1]
It's from you?
https://forum.xda-developers.com/t/help-recovery-unable-to-find-recovery-img-in-motorola-moto-g72-vicky-firmware-zip.4589039/

I see your DT and thatt's not good yet to MT6789 according the specs by your device.

**Comment by sudo-Atharva** [comment]
Yup that's me it's still raw and untested so I acknowledge the problems

**Comment by lopestom** [comment]
> Yup that's me it's still raw and untested so I acknowledge the problems

Your Device Tree is much bad/wrong because you're copying from others recovery/boot DT.
Your device motorola_vicky has header V4 so its in the vendor_boot partition. Not need kernel, boot or recovery partition filesystem.
You need more than ctrlC & ctrlV...................

if you continue with wrong way/decision so you can brick your device.

**Comment by sudo-Atharva** [comment]
Yeah Understand that just was experimenting didn't yet try to build

**Comment by lopestom** [comment]
> > It's from you? https://forum.xda-developers.com/t/help-recovery-unable-to-find-recovery-img-in-motorola-moto-g72-vicky-firmware-zip.4589039/
> > I see your DT and thatt's not good yet to MT6789 according the specs by your device.
> 
> As far as I know my device has mt8781 chipset (helio g99) as seen on gsm-arena website

You have stock ROM?

Using vendor_boot so use https://github.com/osm0sis/Android-Image-Kitchen/tree/AIK-Linux
[v3.8 - add hdr_v3 boot/vendor_boot support, improve lz4 support, fixes](https://github.com/osm0sis/Android-Image-Kitchen/commit/f1ec4343ccf7016fe1c85bb4d3aa9a65bfae0871)
